src/pages/user.vue

```diff
@@ -21,26 +21,18 @@
    </div>
 </template>
 <script>
+import { mapActions } from 'vuex'
 export default{
     data(){
         return{}
     },
     methods:{
-        logoutForm(){
-            this.$Auth().signOut().then(() => {
-                this.$router.push({
-                    path:"login"
-                })
-  
-        }).catch((error) => {
-  
-         });
-        }
+      ...mapActions(["logoutForm"]),
+        signoutUser(){                     //submit handler for user logout//
```

Comment:
@pavankumarhb it should be written before the method starting 
like this 
//Submit handler for user logout
signoutUser(){
and no need to put // at the end